### PR TITLE
KEYCLOAK-15109 Ingress/Route Host Override

### PR DESCRIPTION
**Pull request by jeesmon**

### CODE REVIEW

**Review by slaskawi — DISMISSED** [review]
@jeesmon LGTM! Thank you for submitting this Pull Request.

@andymunro May I kindly ask you to review/merge this?

File: server_installation/topics/operator/keycloak-cr.adoc [hunk 1]
```diff
@@ -28,7 +28,10 @@ The Keycloak custom resource is a YAML file that defines the parameters for inst
 ifeval::[{project_community}==true]
  or an Ingress for Kubernetes
 endif::[]
- for the {project_name} cluster.
+ for the {project_name} cluster. You can set `host` to override automatically chosen host name for Route
```
Comment (andymunro) [inline server_installation/topics/operator/keycloak-cr.adoc]: add "the" after "automatically"

**Review by andymunro — CHANGES_REQUESTED** [review]
thanks, just one change.

File: server_installation/topics/operator/keycloak-cr.adoc (hunk 1 above)
Comment (jeesmon) [reply server_installation/topics/operator/keycloak-cr.adoc]: @andymunro Just want to double check. What we are trying to support is to allow overriding "automatically chosen" host name. Here is the snippet from Openshift Route type.

```
        // host is an alias/DNS that points to the service. Optional.
	// If not specified a route name will typically be automatically
	// chosen.
	// Must follow DNS952 subdomain conventions.
	Host string `json:"host" protobuf:"bytes,1,opt,name=host"`
```

**Comment by jeesmon** [comment]
@andymunro Can you please check my comment when you get a chance? https://github.com/keycloak/keycloak-documentation/pull/1058#discussion_r522636040 Thanks

File: server_installation/topics/operator/keycloak-cr.adoc (hunk 1 above)
Comment (andymunro) [inline server_installation/topics/operator/keycloak-cr.adoc]: Yes, thank you.  I had meant to say add "the" BEFORE "automatically."  thanks for checking.

**Comment by jeesmon** [comment]
@andymunro fixed, thanks

**Review by andymunro — DISMISSED** [review]
Thanks, Jeesmon. Looks good now.

**Review by vmuzikar — DISMISSED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**